src/lib/experimental/fetch/icp.ts

```diff
@@ -61,7 +61,7 @@ export async function fetchIcp(
         }
     );
 
-    const decodedResult = IDL.decode(funcIdl.retTypes, result);
+    const decodedResult = IDL.decode(funcIdl.retTypes, result.buffer);
```

Comment:
I really want to understand this. Let's discuss why this worked before, because it doesn't make sense to me fully why it would not continue to work.